diagnostic.md

```diff
@@ -36,19 +36,19 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
-
+Relative. It is stored locally. Locally stored information is relative.
```

Comment:
absolute.  You could type that path in from anywhere in the file system and it would take you to the same location